netsim/ansible/templates/vlan/cumulus.j2

```diff
@@ -35,7 +34,7 @@ cat >/etc/network/interfaces.d/51-bridge-interfaces.intf <<CONFIG
 auto {{ ifdata.ifname }}
 {%   endif %}
 iface {{ ifdata.ifname }}
-{%   if ifdata.vlan.trunk_id is defined %}
+{%   if ifdata.vlan.trunk_id is defined and ifdata.type == "svi" %}
```

Comment:
This looks fishy. Why would an SVI interface have VLAN trunk ID defined?

Reply:
```if ifdata.vlan.trunk_id is defined and ifdata.type != "lag"```